mmf/configs/zoo/models.yaml

```diff
@@ -1,3 +1,40 @@
+lxmert:
+  pretrained:
+    vqa2:
+      defaults:
+        version: 1.0
+        resources:
+        - url: 
```

Comment:
You will have to add the url of the file in your server from where the models can be downloaded. You can check the other examples in this file.

Reply:
I added the url in my test. I also tried other models like vilbert, but still didn't automatically download their weights.
Is there any thing to specify in the config yaml files?